tensorflow/lite/kernels/internal/reference/pad.h

```diff
@@ -168,6 +168,16 @@ inline void PadImageStyle(const tflite::PadParams& op_params,
       output_data);
 }
 
+template <typename P>
+inline void PadImageStyle(const tflite::PadParams& op_params,
```

Comment:
shall we refactor a little bit that input & output types can be template arguments?